generators/ci-cd/templates/jenkins/Jenkinsfile.ejs

```diff
@@ -184,10 +184,10 @@ node {
 <%= gitLabIndent %>        // A pre-requisite to this step is to setup authentication to the docker registry
     <%_ if (buildToolMaven) { _%>
 <%= gitLabIndent %>        // https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#authentication-methods
-<%= gitLabIndent %>        sh "./mvnw -ntp jib:build"
+<%= gitLabIndent %>        sh "./mvnw -ntp package -Pprod verify jib:build"
```

Comment:
I wonder if we can skip tests at this phase ?

Reply:
You're right, I'll remove that part.